sdk/storage/azure-storage-blob/src/test/java/com/azure/storage/blob/APISpec.groovy

```diff
@@ -189,7 +189,7 @@ class APISpec extends Specification {
     }
 
     static boolean liveMode() {
-        return setupTestMode() == TestMode.RECORD
+        return setupTestMode() == TestMode.RECORD && false
```

Comment:
it looks like a bad hardcoding isn't it?  why bother to call `setupTestMode()` if we are returning false any how
why not just returning false?

Reply:
I was forcing the live test only tests not to record playback records while I was updating the other tests, removed.